lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -533,6 +537,9 @@ def combine_checkbox_answers(record: dict, coded_question: str) -> Optional[List
     if coded_question == 'race':
         return race(answers)
 
+    if re.match(r'fu_[1-4]{1}_symptoms$', coded_question):
```

Comment:
FWIW the `{1}` is unnecessary here because by default the regex will only look for a single match for a character in the square brackets.

Reply:
Thanks, that's good to know 😄